config/config.go

```diff
@@ -138,17 +146,6 @@ func (conf *TerraformExtraArguments) String() string {
 		conf.EnvVars)
 }
 
-// Return the default path to use for the Terragrunt configuration file. The reason this is a method rather than a
-// constant is that older versions of Terragrunt stored configuration in a different file. This method returns the
-// path to the old configuration format if such a file exists and the new format otherwise.
-func DefaultConfigPath(workingDir string) string {
```

Comment:
What was it that required moving this method to another package?

Reply:
As far as I remember to GO compiler complaining about an import loop.

Reply:
Right, I'm just trying to understand which import loop in case there's a way to avoid extra packages, extra refactoring, etc.